### ISSUE TITLE
Cannot deploy app on device

### ISSUE THREAD

**Issue by slavchev** [NativeScript/nativescript-cli#2217]
On some environments `tns run android` and `tns debug android` commands fail with the following message:
```
The new package failed because the current SDK version is older than that required by the package.
```
When inspecting the problem with `--log trace` parameter it turns out that `adb install` fails with `INSTALL_FAILED_OLDER_SDK`. When inspecting the `*.apk` file with `aapt l -a <file.apk>` it shows the following result:
```
...
   E: uses-sdk (line=7)
     A: android:minSdkVersion(0x0101020c)=(type 0x10)0x11
     A: android:targetSdkVersion(0x01010270)="__APILEVEL__" (Raw: "__APILEVEL__")
...
```
In short, the following [placeholder](https://github.com/NativeScript/template-hello-world/blob/b39732b827d6b9fa21f2800d7e020be967527a55/App_Resources/Android/AndroidManifest.xml#L15) in `AndroidManifest.xml` is not replaced.

**Comment by ecavalcanti** [comment]
Works if you run before: tns build android

**Comment by tzraikov** [comment]
This should be addressed when using CLI 2.4.x. Can you describe the exact environment and steps to reproduce the issue.

**Comment by Plamen5kov** [comment]
Cannot reproduce.
